src/contracts/middleware/IndexRegistry.sol

```diff
@@ -141,6 +134,20 @@ contract IndexRegistry is IIndexRegistry {
         return operatorIndexToCheck.index;
     }
 
+    /// @notice Returns an ordered list of operators of the services for the given `quorumNumber` at the given `blockNumber`
+    function getOperatorListForQuorumAtBlockNumber(uint8 quorumNumber, uint32 blockNumber) external view returns (bytes32[] memory){
+        bytes32[] memory quorumOperatorList = new bytes32[](_getTotalOperatorsForQuorumAtBlockNumber(quorumNumber, blockNumber));
+        for (uint i = 0; i < globalOperatorList.length; i++) {
+            bytes32 operatorId = globalOperatorList[i];
+            uint32 index = _getIndexOfOperatorForQuorumAtBlockNumber(operatorId, quorumNumber, blockNumber);
+            // if the operator was not in the quorum at the given block number, skip it
+            if(index == type(uint32).max)
+                continue;
+            quorumOperatorList[index] = operatorId;
```

Comment:
we need to make sure that this behavior -- specifically, returning the max uint32 from `_getIndexOfOperatorForQuorumAtBlockNumber` when the operator was not in the quorum at the given block number -- is super clearly documented!

Reply:
agreed